ionic-angular-template/src/auth/sign-in/sign-in.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, OnInit } from '@angular/core';
+import { NgForm, FormGroup, FormBuilder, Validators, FormControl } from '@angular/forms';
+import { AuthService } from '../../services/auth.service';
+import { Router } from '@angular/router';
+import { AlertController } from '@ionic/angular';
+
+@Component({
+  selector: 'app-sign-in',
+  templateUrl: './sign-in.component.html',
+  styleUrls: ['./sign-in.component.scss'],
+})
+export class SignInComponent implements OnInit {
+  public signInForm: FormGroup;
+  public email: FormControl;
+  public password: FormControl;
+  constructor(
+    private authService: AuthService,
+    private router: Router,
+    private alertCtrl: AlertController,
+    private fb: FormBuilder
+  ) { }
+
+  ngOnInit() {
+    this.email = new FormControl('', Validators.compose([Validators.required, Validators.email]));
+    this.password = new FormControl('', Validators.compose([Validators.required, Validators.minLength(6)]));
+    this.signInForm = this.fb.group({
+      email: this.email,
+      password: this.password
+    });
+  }
+
+  async signIn(email: string, password: string) {
```

Comment:
You can get the values of the form from the getters. You can implement getter for every formControlName.